src/accessibility-statement.md

```diff
@@ -58,42 +58,34 @@ The team does not have any content within these products outside the scope of th
 
 ### Other identified and tracked accessibility concerns
 
-In addition to non-compliant content, the team also track additional concerns about accessibility which:
+In addition to non-compliant content, the team also tracks additional concerns about accessibility which:
 
 - show strong evidence of being accessibility barriers
-- do not constitute a failure in one or more WCAG 2.1 Level AA criteria
+- do not constitute a failure in any WCAG 2.2 Level AA criteria
 - are not classified as non-compliance with the accessibility regulations
 - are determined as worth featuring in this accessibility statement
 
-The team documents these additional concerns in two repositories under the ‘accessibility-concern’ issue label:
+The team documents these additional concerns in two repositories under the ‘accessibility concern’ issue label:
 
-- [GOV.UK Design System website label for accessibility concerns](https://github.com/alphagov/govuk-design-system/labels/accessibility-concern)
-- [GOV.UK Frontend codebase label for accessibility concerns](https://github.com/alphagov/govuk-frontend/labels/accessibility-concern)
+- [GOV.UK Design System website label for accessibility concerns](https://github.com/alphagov/govuk-design-system/issues?q=is%3Aopen+is%3Aissue+label%3A%22accessibility+concern%22)
+- [GOV.UK Frontend codebase label for accessibility concerns](https://github.com/alphagov/govuk-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22accessibility+concern%22)
 
-For the GOV.UK Design System website and the GOV.UK Frontend Documentation website, the team do not currently know of any additional accessibility concerns to add to this statement.
+For the GOV.UK Design System website and the GOV.UK Frontend Documentation website, the team does not currently know of any additional accessibility concerns to add to this statement.
 
 Additional accessibility concerns from the GOV.UK Frontend codebase include:
 
```

Comment:
As mentioned in an internal chat, the two GitHub issues listed here say to "track our progress". But both of those issues are closed and there is not much to track. The only way to track is via looking at all the issues that are linking to it, so not very user-friendly. We should maybe try to adjust the language (and therefore expectations), or change the links to point to an issues that explains more and really tracks it.